docs/pages/workflows/jobs.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Jobs
+description: Learn which pre-packaged and custom jobs you can run with Workflows.
+---
+
+Workflows sequence jobs together to automate your development and release processes. You can run two types of jobs: pre-packaged jobs and custom jobs.
+
+Pre-packaged jobs are provided by Expo and meet common use cases for developers, like building, submitting, and running end to end test. Since they are provided by Expo, they are continuously maintained and updated.
```

Comment:
```suggestion
Expo provides pre-packaged jobs that meet common use cases for developers, such as building, submitting, and running end-to-end tests. Since they are provided by Expo, they are continuously maintained and updated.
```